radixdlt-java/src/main/java/com/radixdlt/client/dapps/wallet/RadixWallet.java

```diff
@@ -107,7 +107,7 @@ public SendResult send(
 		@NonNull RadixAddress toAddress
 	) {
 		Objects.requireNonNull(toAddress, "toAddress must be non-null");
-		return this.send(amount, toAddress, null);
+		return this.send(amount, null, toAddress);
```

Comment:
Changed the ordering of params for the send method therefore sendWhenAvailable should also be changed for consistency.

Reply:
Good point, fixed.